test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -644,21 +619,7 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
                     return Task.CompletedTask;
                 }, receiveTcs);
 
-                connection.Closed += e =>
```

Comment:
Don't we still need this functionality? Not sure we can just delete it.

Reply:
Added this back. Not being marked as resolved though